platform/broadcom/sonic-platform-modules-accton/as7712-32x/cfg/th-as7712-32x100G.config.bcm

```diff
@@ -0,0 +1,295 @@
+# accton_as7710_32x 32x100G SDK config
```

Comment:
yes. I received it. please remove the file here. I will update the saibcm package.